m2m-transfers/V3/REPWRITERSPECS/BANNO.M2MTRANSFERS.V3.POW

```diff
@@ -93,6 +93,8 @@
 **   Ver. 3.2.6  12/11/2024 T. Kainz - Banno
 **               * Corrected issue where program was erring when no transfer record FM
 **                 history was found indicating the record creation date
+**   Ver. 3.3.0  02/04/2025 R. Robison - Banno
+**               * Adding support for errorDisplayMessage to accomodate memo mode message
```

Comment:
Hello @rrobison44 , Only thing I see so far is a typo ,accomodate, on line 97 of BANNO.MTMTRANSFERS.V3.POW